docs/docsite/rst/dev_guide/developing_modules.rst

```diff
@@ -1,110 +1,63 @@
 .. _developing_modules:
-
-Developing Modules
-==================
-
-.. contents:: Topics
-
-.. _module_dev_welcome:
-
-Welcome
-```````
-This section discusses how to develop, debug, review, and test modules.
-
-
-Ansible modules are reusable, standalone scripts that can be used by the Ansible API,
-or by the :command:`ansible` or :command:`ansible-playbook` programs.  They
-return information to ansible by printing a JSON string to stdout before
-exiting.  They take arguments in one of several ways which we'll go into
-as we work through this tutorial.
-
-See :ref:`all_modules` for a list of existing modules.
-
-Modules can be written in any language and are found in the path specified
-by :envvar:`ANSIBLE_LIBRARY` or the ``--module-path`` command line option or
-in the :envvar:`library section of the Ansible configuration file <ANSIBLE_LIBRARY>`.
-
 .. _module_dev_should_you:
 
+****************************
 Should You Develop A Module?
-````````````````````````````
-Before diving into the work of creating a new module, you should think about whether you actually *should*
-develop a module. Ask the following questions:
+****************************
+
+Developing Ansible modules is easy, but often it isn't necessary. Before you start writing a new module, ask:
 
 1. Does a similar module already exist?
 
-There are a lot of existing modules available, you should check out the list of existing modules at :ref:`modules`
+An existing module may cover the functionality you want. Ansible Core includes thousands of modules. Search our :ref:`list of existing modules <all_modules>` to see if there's a module that does what you need.
 
-2. Has someone already worked on a similar Pull Request?
+2. Does a Pull Request already exist?
 
-It's possible that someone has already started developing a similar PR. There are a few ways to find open module Pull Requests:
+An existing Pull Request may cover the functionality you want. If someone else has already started developing a similar module, you can review and test it. There are a few ways to find open module Pull Requests:
 
 * `GitHub new module PRs <https://github.com/ansible/ansible/labels/new_module>`_
 * `All updates to modules <https://github.com/ansible/ansible/labels/module>`_
 * `New module PRs listed by directory <https://ansible.sivel.net/pr/byfile.html>`_ search for `lib/ansible/modules/`
 
-If you find an existing PR that looks like it addresses the issue you are trying to solve, please provide feedback on the PR -  this will speed up getting the PR merged.
+If you find an existing PR that looks like it addresses your needs, please provide feedback on the PR -  this will speed up the review and merge process.
```

Comment:
There is an extra space after the -. Also, change this to an em-dash: 'PR — this will...'.